packages/pass-style/src/types.js

```diff
@@ -69,14 +69,14 @@ export {};
  */
 
 /**
- * @typedef {Passable} Remotable
+ * @typedef {Passable} RemotableObject
```

Comment:
Ugh the conflict with the also exported `Remotable` runtime value from `make-far.js` is unfortunate. Maybe we can fix this mess once we have more precise types for `Passable`.

Reply:
Agreed,  I'm just happy to have a workaround until then. :)